kuzzle/disconnect_test.go

```diff
@@ -12,17 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package kuzzle_test
+package kuzzle
```

Comment:
Why did you update the package name?

Reply:
So we are in the same package as Kuzzle and don't need to import the kuzzle package.

Reply:
Use different package for unit testing is useful to test integration behavior. But ok :slightly_smiling_face:

Reply:
Keeping the same package name for tests is idiomatic to Go, it's meant to be used like that.